proto/decentraland/social_service_v2/social_service.proto

```diff
@@ -88,6 +130,31 @@ message FriendshipUpdate {
   }
 }
 
+message GetFriendshipStatusPayload {
+  User user = 1;
+}
+
+enum FriendshipStatus {
+  REQUEST_SENT = 0;
+  REQUEST_RECEIVED = 1;
+  CANCELED = 2;
+  ACCEPTED = 3;
+  REJECTED = 4;
+  DELETED = 5;
+  BLOCKED = 6;
+}
+
+message GetFriendshipStatusResponse {
+  message Ok {
+  FriendshipStatus status = 1;
+  optional string message = 2;
+  }
+  oneof response {
+    Ok accepted = 1;
+    InternalServerError internal_server_error = 2;
+  }
+}
+
 service SocialService {  
```

Comment:
Shall we delete this service as you have created `SocialServiceV2`?

Reply:
I've marked it as deprecated, as suggested.